commcare-cloud-bootstrap/environment/private.yml.j2

```diff
@@ -48,3 +48,4 @@ localsettings_private:
   SECRET_KEY: '{{ generate_uuid() }}'
 
 ansible_sudo_pass: '{{ generate_uuid() }}'
+DETAILED_TAGGING_DOMAINS: ''
```

Comment:
@dannyroberts I'm not sure if this is necessary to be able to reference the variable in other files, like in the `public.yml` file below.